src/Components/SysTable/constants.test.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { mergedColumns } from './constants';
+
+const intl = {
+  formatMessage: jest.fn((msg) => msg.defaultMessage),
+};
+
+const messages = {
+  name: { defaultMessage: 'Name' },
+  dataNotAvailable: { defaultMessage: 'Data not available' },
+  lastStatus: { defaultMessage: 'Last Status' },
+  lastMatched: { defaultMessage: 'Last Matched' },
+  noHostHas: { defaultMessage: 'No Host Has' },
+  never: { defaultMessage: 'Never' },
+  totalMatches: { defaultMessage: 'Total Matches' },
+  totalMatchesNote: { defaultMessage: 'Total Matches Note' },
+  lastScan: { defaultMessage: 'Last Scan' },
+};
+
+const InsightsLink = ({ to, children }) => <a href={to}>{children}</a>;
+const StatusLabel = ({ isDisabled, hasMatch }) => (
+  <span>{isDisabled ? 'Disabled' : hasMatch ? 'Matched' : 'No Match'}</span>
+);
+const Tooltip = ({ content, children }) => (
+  <span title={content}>{children}</span>
+);
+const DateFormat = ({ date }) => <time>{date.toLocaleString()}</time>;
+const cellWidth = (width) => width;
+const info = ({ tooltip }) => tooltip;
+
+const columns = [
```

Comment:
How can we avoid duplicating these?

Reply:
I can remove them if I update the expected values correctly. Did that and updated the tests